phpunit.xml.dist

```diff
@@ -33,7 +33,7 @@
         </exclude>
     </source>
 
-    <extensions>
-        <bootstrap class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension" />
-    </extensions>
+    <!--<extensions>
```

Comment:
Disable `extensions` element. PHPUnit v10 doesn't have this element.  If this is causing issues with functional tests, then we need to down grade to phpunit v9 again.

Reply:
I just merged #396 and the updated extension block from that patch is already updated for phpunit 10 [according to the test bundle docs](https://github.com/dmaicher/doctrine-test-bundle#using-the-bundle-with-phpunit), so there shouldn't be need to disable this top level `extensions` block

Reply:
Ow let me check again.